docs/capabilities/app-client.md

```diff
@@ -123,13 +123,13 @@ const { result, app } = factory.create({
   populateAppCallResources: true,
 })
 // Specify parameters for ABI method call
-const { result, app } = factory.create({
+const { result, appClient } = factory.send.create({
   method: 'create_application',
   args: [1, 'something'],
 })
 ```
 
-If you want to construct a custom create call, use the underlying [`algorand.send.appCreate` / `algorand.transactions.appCreate` / `algorand.send.appCreateMethodCall` / `algorand.transactions.appCreateMethodCall` methods](./app.md#creation) then you can get params objects:
+If you want to construct a custom create call, use the underlying [`algorand.send.appCreate` / `algorand.createTransaction.appCreate` / `algorand.send.appCreateMethodCall` / `algorand.createTransaction.appCreateMethodCall` methods](./app.md#creation) then you can get params objects:
```

Comment:
`transactions` -> `createTransaction` to get the verb plus the consistency of singular rather than plural.

We **didn't** rename `params` because the _correct_ name would be `getTransactionParams` and it's simply too verbose and the places you are most likely to use params are inline when nesting ABI method arguments where you want less verbosity. As such we updated the jsdoc comment to make it more obvious how it's used and feel it's one that people will follow examples to understand how to use (and honestly, it's a slightly more advanced use case anyway so I think it's OK if it's a bit less discoverable).